apps/meteor/package.json

```
@@ -73,6 +73,7 @@
 		"@bugsnag/js": "~7.20.2",
 		"@bugsnag/plugin-react": "~7.19.0",
 		"@datastructures-js/priority-queue": "^6.3.5",
+		"@date-fns/tz": "^1.4.1",
```

Biome (2.4.10)

[error] The key `@date-fns/tz` was already declared.

(lint/suspicious/noDuplicateObjectKeys)